dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/service/SoftwareService.java

```diff
@@ -41,50 +41,83 @@
       "groupId", "org.icgc.dcc",
       "artifactId", "icgc-storage-client",
       "classifier", "dist",
-      "repository", "dcc-release");
+      "repository", "dcc-release"
+      );
+  private static final Map<String, String> ICGCPARAMS = ImmutableMap.of(
```

Comment:
On second thought, let's move this to `ServerProperties` as `SoftwareProperties` class. We can default the bean to have these values, but still enable the config file to override them. Ping me tomorrow and I'll explain further.